_benchmark/glossary.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+title: Glossary
+nav_order: 10
+---
+
+# OpenSearch Benchmark glossary
+
+The following terms are commonly used in OpenSearch Benchmark:
+
+- **Corpora**: A collection of documents.
```

Comment:
Line 12: "equivalent to" => "equal to"? ("equal" meaning "same in all respects," "equivalent" meaning "similar but not identical")